pkgs/by-name/de/debian-archive-keyring/package.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, gnupg
+, jetring
+, debian-archive-keyring
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "debian-archive-keyring";
+  version = "2023.4";
+
+  src = fetchFromGitLab {
+    domain = "salsa.debian.org";
+    owner = "release-team";
+    repo = "${finalAttrs.pname}";
```

Do not parameterize pname.

```suggestion
    repo = "debian-archive-keyring";
```